FLAnimatedImage/FLAnimatedImageView.m

```diff
@@ -87,6 +87,12 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
 - (void)commonInit
 {
     self.runLoopMode = [[self class] defaultRunLoopMode];
+    
+#ifdef __IPHONE_11_0
```

Comment:
Would it be safe to drop `#ifdef __IPHONE_11_0` at this stage? Most people should probably be on Xcode 9 by now I imagine. And not everyone will get the latest framework version immediately anyway.